src/test/ui/str-as-char.fixed

```diff
@@ -0,0 +1,14 @@
+// Copyright 2017 The Rust Project Developers. See the COPYRIGHT
+// file at the top-level directory of this distribution and at
+// http://rust-lang.org/COPYRIGHT.
+//
+// Licensed under the Apache License, Version 2.0 <LICENSE-APACHE or
+// http://www.apache.org/licenses/LICENSE-2.0> or the MIT license
+// <LICENSE-MIT or http://opensource.org/licenses/MIT>, at your
+// option. This file may not be copied, modified, or distributed
+// except according to those terms.
+
+fn main() {
+    println!("●●");
+    //~^ ERROR character literal may only contain one codepoint
```

Comment:
kinda wish we could filter these out entirely but this seems annoying

(I'm not quite sure why we require `//~^ ERROR` on ui tests anyway given that it's all in the output)